management/virtualmachine/client.go

```diff
@@ -209,7 +209,7 @@ func (vm VirtualMachineClient) StartRole(cloudServiceName, deploymentName, roleN
 	return vm.client.SendAzurePostRequest(requestURL, startRoleOperationBytes)
 }
 
-func (vm VirtualMachineClient) ShutdownRole(cloudServiceName, deploymentName, roleName string) (management.OperationID, error) {
+func (vm VirtualMachineClient) ShutdownRole(cloudServiceName, deploymentName, roleName string, postactions ...PostShutdownAction) (management.OperationID, error) {
```

Comment:
You don't have to make this argument variadic. We don't have much API breaking concerns for now. Feel free to make it a single argument as you're using only the first element.